aptos-move/aptos-vm-types/src/resolver.rs

```diff
@@ -283,3 +283,51 @@ pub trait StateValueMetadataResolver {
         state_key: &StateKey,
     ) -> anyhow::Result<Option<StateValueMetadataKind>>;
 }
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum ResourceGroupSize {
```

Comment:
Either here or at get, could you add a brief description for the Combined logic?